wasm2c/wasm-rt.h

```diff
@@ -47,6 +47,8 @@ extern "C" {
 #if defined(__linux__) || defined(__unix__) || defined(__APPLE__)
 #if defined(__WORDSIZE) && __WORDSIZE != 64
 #warning "Signal handler is only supported on 64-bit architectures"
+#undef WASM_RT_MEMCHECK_SIGNAL_HANDLER
+#define WASM_RT_MEMCHECK_SIGNAL_HANDLER 0
```

Comment:
Perhaps better not to enable it in the first place just above?

Reply:
Yeah, but that's only done if it's not already defined, so it could be defined externally too. Though in that case, I guess we'd want to error out instead of warn.

Reply:
I think an error would make more sense yeah.  `undef` seems little odd.

Obviously this just a nit.